### DISCUSSION TITLE
Feedback /Обратная связь

### DISCUSSION THREAD

**Discussion by SolitarySpiral** [SolitarySpiral/python-image-dowloader#20] [General]
Если вам данное ПО оказалось полезным, я бы хотел получить от вас обратную связь. Возможно у вас есть предложения, пожелания или замечания. Ну или просто хотите поблагодарить автора за старания. / If this software turned out to be useful to you, I would like to get feedback from you. Perhaps you have suggestions, wishes or comments. Well, or you just want to thank the author for his efforts.

**Comment by Barkid123** [comment]
ПО рабочее и в целом хорошее, мне понравилось (да я тот самый единственный папищек, автор). Но есть одна проблемка, что оно отказывается работать с nozomi.la. Ну, установил всё правильно, всё запускается, теги работают, но только с rule34.xxx. 
![WindowsTerminal_vzQdP7aPZ7](https://github.com/user-attachments/assets/511ea98b-b990-4d24-8128-73f34070705d)
Скриншот пример работы с rule34.xxx

С нозоми же вот такое
![image](https://github.com/user-attachments/assets/62186172-cdc4-4aeb-988b-b8c65e067f64)

Помимо этого ругается на строку 679 в main.py
![image](https://github.com/user-attachments/assets/100fb9bf-cd13-4794-8901-dcc2288f2d2a)

Логи остановки программы:
[ошибка.txt](https://github.com/user-attachments/files/20187800/default.txt)

**Reply by SolitarySpiral** [reply]
Привет. р34 у меня работает нормально. Не смотря на то, что есть косяк в post_ids
![image](https://github.com/user-attachments/assets/ce373a3d-4722-4035-816e-4b9cdafbb502)
Что же до нозоми, у меня на сайте вообще не прогружаются данные постов.
![image](https://github.com/user-attachments/assets/c0b85d5f-8961-481f-8f43-f5c938a81695)
Нет данных - нечего скачивать. Вероятно запросы заграницу блокируются провайдером, даже прокси расширение для браузера или unicorn не помогают.

**Reply by Barkid123** [reply]
Тогда если дело в блокировке, то у меня nozomi проксируется через нидерланды и как бы непосредственный доступ к сайту со всем его просматриваемым содержимым без входа в учётку я имею безграничный. Ну и тут ограничение уже от самого сайта с его попытками крашнуть мне браузер на ПК и более успешной крашнуть на мобилке при попытке сохранить какой-то ресурс с сайта. Ну, благо, браузер через лютый пердёж примерно в 1-3 минуты одупляется и у меня появляется контекстное меню "сохранить". Но собственно сохранять так каждый файл, ожидая ~2 минуты ну очень некомфортно. И вот назревает предложение, почему бы не сделать совместимость приложения с таким китайским прокси как NekoRay (https://github.com/MatsuriDayo/nekoray), ну либо ещё как. Например, я использую NekoRay для связи ВПС-сервера с компом и уже через расширение SwitchyOmega связываю это дело с браузером, чтобы у меня проксировались сайты (либо все, либо выборочно). Я в этой теме на самом деле не очень шарю, как и в питоне и вот кратко изложил принцип работы сети, к которой я подключён. Ну, в любом случае, даже если не через совместимость, то, возможно, через какой-то встроенный dpi канал
